MSAL/test/integration/native_auth/end_to_end/reset_password/MSALNativeAuthResetPasswordEndToEndTests.swift

```diff
@@ -61,23 +64,25 @@ final class MSALNativeAuthResetPasswordEndToEndTests: MSALNativeAuthEndToEndBase
         let resetPasswordCompletedExp = expectation(description: "reset password completed")
         let resetPasswordRequiredDelegate = ResetPasswordRequiredDelegateSpy(expectation: resetPasswordCompletedExp)
 
-        resetPasswordVerifyDelegate.newPasswordRequiredState?.submitPassword(password: "password", delegate: resetPasswordRequiredDelegate)
+        let uniquePassword = "password.\(Date().timeIntervalSince1970)"
```

Comment:
This should use @diegojerezba [method](https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/2261/files) -  `generateRandomPassword`

Reply:
Done.